dockerfiles/init/manifests/che.env

```diff
@@ -429,6 +429,8 @@ CHE_SINGLE_PORT=false
 #     property 'che.workspace.auto_snapshot=false'.
 #CHE_WORKSPACE_AUTO__SNAPSHOT=false
 
+#SYSTEM_SUPER_PRIVILEGED_MODE=false
```

Comment:
I'd add a description for this property. See https://github.com/codenvy/codenvy/blob/master/dockerfiles/init/manifests/codenvy.env#L40